pandas/tseries/tools.py

```diff
@@ -222,6 +222,11 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         datetime strings, and if it can be inferred, switch to a faster
         method of parsing them. In some cases this can increase the parsing
         speed by ~5-10x.
+    origin : scalar convertible to Timestamp / string ('julian', 'epoch'), default 'epoch'
+        Define relative offset for the returned dates.
+        - If 'epoch', offset is set to 1-1-1970
```

Comment:
Can you leave a blank line between "Define ..." and "- If .." (needed for correct formatting of the restructured text)